Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -195,8 +195,8 @@ export default function ModelExperimental(options) {
   this._id = options.id;
   this._idDirty = false;
 
-  const color = options.color;
-  this._color = defined(color) ? Color.clone(color) : undefined;
+  const color = defaultValue(options.color, Color.WHITE);
```

Comment:
I'm guessing this is why the `3D Tiles Next Photogrammetry` example looks a bit washed-out now?

![image](https://user-images.githubusercontent.com/8422414/181366018-eedf15bf-8e9c-4f23-8adb-4f73f36fda33.png)

Is this what it's supposed to look like, and if so, I wonder if we should tweak the settings in the Sandcastle to look a bit more vibrant

Reply:
Does `color` need to default to white? That will cause blending to happen when the mode is mix or replace. And the `ModelColorPipelineStage` will always be added.

It's ok if `Model` and `ModelExperimental` deviate a little bit here. I assume `Model` treats white as a special case where it disables blending, but it's a little nicer if `color = undefined` means no blending.

Reply:
Ah, I just uploaded a fix that disables the `ModelColorStage` if CPU styling is being used.

The bug was happening because `Model`'s default color was `Color.WHITE`, and the shader enabled the `MIX` blend mode so the styling colors would look more blended with the model. But this was making it also blend the color for `ModelExperimental`, so it looked paler than usual.

Should I revert to a default `undefined` color for `ModelExperimental`?

Reply:
> Should I revert to a default `undefined` color for `ModelExperimental`?

Yeah, I think that's the best thing to do